cmd/setup-gh.go

```diff
@@ -231,18 +233,26 @@ func getCloudProvider() string {
 	return selectResponse
 }
 
-func GetAzSubscriptionId(subIds []string) string {
+func GetAzSubscriptionId(subLabels []providers.SubLabel) string {
 	selection := &promptui.Select{
```

Comment:
just a general idea: I like making these more generic so we can use them in multiple areas.

https://github.com/Azure/aks-spin-plugin/blob/main/pkg/prompt/select.go#L17